terraform/cluster/aws/main.tf

```diff
@@ -278,6 +278,42 @@ resource "aws_launch_template" "cluster" {
     }
   }
 
+  user_data = base64encode(<<-EOF
```

Comment:
use terraform template file instead

also this will trigger update for all the existing cluster

can you change it as we discussed like if user explicitly don't set it then no need to provide it as userdata?